packages/web-api/src/controllers/get-report-controller.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { getSarifReportMock } from '../../get-report-func/sarif-report-mock';
```

Comment:
controllers should not interact with func folder. but it is fine if you don't want to fix this since we will be removing in the next feature

Reply:
updated